README.md

```diff
@@ -80,76 +88,84 @@ When you update `package.json` to include new dependencies, or when you pull cha
 ```
 
 <br/>
 
-## **Configuring Zowe CLI**
-
-Zowe CLI configuration is made up of different **profiles**. The profiles contain the information that Zowe CLI needs to communicate with the mainframe system. For example, credentials and z/OSMF host name. If you try to use Zowe CLI functionality and you get an error message that Zowe CLI failed to load any profiles, see the `zowe profiles create --help` command for the group of commands that you are trying to use (if any) to initialize your configuration.
+## Configuring Zowe CLI
 
-The most fundamental Zowe CLI profile is a `zosmf` profile. Issue the following command to understand how to create a `zosmf` profile in Zowe CLI:
+Zowe CLI team configuration is made up of different **profiles**. Each profile contains the information that Zowe CLI needs to communicate with the mainframe system, such as credentials and host name.
 
-```
-zowe profiles create zosmf-profile --help
-```
+The most fundamental Zowe CLI profile is a `zosmf` profile, and it is included when Zowe CLI initializes your team configuration. However, you must still add your specific connection information to complete the `zosmf` profile. To do so, update your `~/.zowe/zowe.config.json` configuration file with a text editor or an IDE (such as Visual Studio Code) on your computer.
 
-After you create your profile, you can confirm that the properties of your profile can connect to and communicate with your mainframe system successfully by issuing the following command:
+After you create and/or finalize your profile, confirm that the properties of your profile can connect to and communicate with your mainframe system successfully:
 
 ```
 zowe zosmf check status
 ```
 
-For detailed information about creating service profiles, creating base profiles, or integrating with Zowe API ML, see [Using Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-using-usingcli/).
+For detailed information about creating profiles, or integrating with Zowe API ML, see the documentation in the [Using Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-using-usingcli/) section of Zowe Docs.
 
-**Tip:** 
-- When you confirm that your profile connects to and communicates with your mainframe system successfully, you can issue the same command at any time to verify the availability and status of the z/OSMF subsystem on your mainframe.
+**Tip:** When you confirm that your profile connects to and communicates with your mainframe system successfully, issue the same command at any time to verify the availability and status of the z/OSMF subsystem on your mainframe.
 
 <br/>
 
-## **Zowe Node Client SDK**
+## Troubleshooting Zowe CLI
+
+If you try to use Zowe CLI functionality and you get an error message that Zowe CLI failed to load any profiles, try issuing the following commands:
+
+- `zowe config report-env` to generate a report on the status of the key areas in your working environment. Address any problems indicated in the report.
```

Comment:
Looks great! thank you for including multiple options! And for cleaning up this page in general :)